examples/shapes_and_collections/hatch_style_reference.py

```diff
@@ -9,11 +9,45 @@
 import matplotlib.pyplot as plt
 from matplotlib.patches import Rectangle
 
-fig, axs = plt.subplots(2, 5, constrained_layout=True, figsize=(10, 5))
+fig, axs = plt.subplots(2, 5, constrained_layout=True, figsize=(9, 4.5))
 
 hatches = ['/', '\\', '|', '-', '+', 'x', 'o', 'O', '.', '*']
 
 
+def hatches_plot(ax, h):
+    ax.add_patch(Rectangle((0, 0), 2, 2, fill=False, hatch=h))
+    ax.text(1, -0.5, f"' {h} '", size=15, ha="center")
+    ax.axis('equal')
+    ax.axis('off')
+
+for ax, h in zip(axs.flat, hatches):
+    hatches_plot(ax, h)
+
+###############################################################################
+# Hatching patterns can be repeated to increase the density.
+
+fig, axs = plt.subplots(2, 5, constrained_layout=True, figsize=(9, 4.5))
```

Comment:
These figures are too big for what they shows.  Suggest just use the normal width and shorter height.

Reply:
How can I view the page that is created by Sphinx? I tried [building the docs](https://matplotlib.org/devdocs/devel/documenting_mpl.html#building-the-docs), but my computer froze up. Is that the only way? Or can you build just one page?

Reply:
The Circle CI artifacts are created below and you can click through on those.  

Its not trivial to build an individual page, and yes building the docs locally is a pain to get set up.  You can, of course, just run the example locally, but you won't see the actual html.

But unless there is a good reason, its probably better to keep the width of the figure the default 6.4in.  And these swatches do not need to be very large.